up42/workflow.py

```diff
@@ -327,7 +327,7 @@ def construct_parameters(
                 input_parameters[data_block_name]["limit"] = len(scene_ids)
                 input_parameters[data_block_name].pop("time")
             elif start_date is not None and end_date is not None:
-                datetime = f"{start_date}T00:00:00Z/{end_date}T00:00:00Z"
+                datetime = f"{start_date}T00:00:00Z/{end_date}T23:59:59Z"
```

Comment:
Although there is no `datetime` module import but minor suggestion would be to name the variable different from module name
```suggestion
                in_datetime = f"{start_date}T00:00:00Z/{end_date}T23:59:59Z"
```

Reply:
renamed